ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -287,13 +287,13 @@ periodics:
       - mountPath: /home/mirror/.docker/config.json
         name: push
         readOnly: true
-        subPath: .dockerconfigjson
+        subPath: config.json
       - mountPath: /etc/imagemirror
         name: config
     volumes:
     - name: push
       secret:
-        secretName: registry-push
+        secretName: registry-push-credentials-quay.io
```

Comment:
The `secret/registry-push` is not maintained by the secret tool.

It contains 2 keys:

```
oc --context api.ci get secret -n ci ${secret} -o yaml | yq -r '.data.".dockerconfigjson"' |base64 -d | jq -r ".auths | keys[]"
https://index.docker.io/v1/
quay.io
```

The one for quay.io has the same value as `secret/registry-push-credentials-quay.io` and the one for docker.io is not used by this job.

It is used in a `buildConfig` on api.ci: We have not started handle those objects yet.
https://github.com/openshift/release/blob/7deb5498d60874387ba5011fb553a78c68062896/projects/kubernetes/node-problem-detector.yaml#L47-L51